example/electric-vehicles.jv

```diff
@@ -51,7 +56,7 @@ pipeline ElectricVehiclesPipeline {
       "VIN (1-10)" oftype VehicleIdentificationNumber10,
       "County" oftype text,
       "City" oftype text,
-      "State" oftype UsStateCode,
+      "State" oftype UsStateCode, // We can just use the element as if it was defined in this file.
```

Comment:
```suggestion
      "State" oftype UsStateCode, // We can just the element as if it was defined in this file.
```

Reply:
I don't get that suggestion :sweat_smile:

Reply:
Me neither, wtf is that guy saying. I'd just ignore him.

Reply:
Haha :laughing: